pages.es/common/act.md

```diff
@@ -0,0 +1,28 @@
+# act
+
+> Ejecuta acciones de GitHub localmente mediante Docker.
+> Más información: <https://github.com/nektos/act>.
+
+- Lista de acciones disponibles:
+
+`act -l`
+
+- Ejecuta el evento por defecto:
+
+`act`
+
+- Ejecuta un evento específico:
+
+`act {{event_type}}`
+
+- Ejecuta una acción específica:
+
+`act -a {{action_id}}`
+
+- No ejecutar actualmente las acciones (p.e. una ejecución en seco):
```

Comment:
`currently` -> `actualmente`, `actually` -> `en realidad`, IMO the best translation for `dry run` is `simulacro`.
```suggestion
- No ejecuta las acciones (p.e. simular una ejecución):
```
`No ejecuta las acciones en realidad` doesn't sound good.